smartsim/_core/_cli/cli.py

```diff
@@ -62,7 +62,9 @@ def __init__(self, menu: t.List[MenuItemConfig]) -> None:
         )
 
         self.register_menu_items(menu)
-        self.register_menu_items([plugin() for plugin in plugins])
+        # Register plugin menu items (currently empty since all plugins were removed)
+        plugin_items: t.List[MenuItemConfig] = [plugin() for plugin in plugins]
+        self.register_menu_items(plugin_items)
```

Comment:
Nit: Now that we require Python >= 3.9, prefer `list[T]` over the (now deprecated) `typing.List[T]`.

```suggestion
        # Register plugin menu items (currently empty since all plugins were removed)
        plugin_items: list[MenuItemConfig] = [plugin() for plugin in plugins]
        self.register_menu_items(plugin_items)
```

Or, better yet, drop the type-hint entirely and let the checker figure it out!

```suggestion
        # Register plugin menu items (currently empty since all plugins were removed)
        plugin_items = [plugin() for plugin in plugins]
        self.register_menu_items(plugin_items)
```